src/main/webapp/help-EventsFilter.html

```diff
@@ -0,0 +1,8 @@
+Filters events that are queued from the gerrit stream events.
+<br>
+If only a few events are intresting to you and you want to reduce load on the queue, then you can control the filter with this setting.
+<br><br>
+<b>Warning</b>
+<br>
+This does not modify the event list in the job configuration view.
+If you use an event from that list that is filtered out by this setting then that trigger will not work.
```

Comment:
A nice future feature could possibly be to automatically filter based on what jobs are configured to trigger on